containers/build_and_push.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -x
+CURRENT_GIT_HASH=$(git rev-parse HEAD)
+PROJECT_ROOT=$(git rev-parse --show-toplevel)
+
+pushd $PROJECT_ROOT
```

Comment:
please quote `$PROJECT_ROOT`, otherwise it will fail if the current path contains a space